vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/Dashboard.java

```diff
@@ -143,6 +143,28 @@ public void removeAll() {
         updateClient();
     }
 
+    /**
+     * Returns maximum column count of the dashboard.
+     *
+     * @return the maximum column count of the dashboard
+     */
+    public Integer getMaximumColumnCount() {
+        String maxColCount = getStyle().get("--vaadin-dashboard-col-max-count");
+        return maxColCount == null ? null : Integer.valueOf(maxColCount);
+    }
+
+    /**
+     * Sets the maximum column count of the dashboard.
+     *
+     * @param maxCount
+     *            the new maximum column count. Pass in {@code null} to set the
+     *            maximum column count back to the default value.
+     */
+    public void setMaximumColumnCount(Integer maxCount) {
+        getStyle().set("--vaadin-dashboard-col-max-count",
+                maxCount == null ? null : String.valueOf(maxCount));
+    }
```

Comment:
nit: to align the getter and setter

```suggestion
    public void setMaximumColumnCount(Integer maxColCount) {
        getStyle().set("--vaadin-dashboard-col-max-count",
                maxColCount == null ? null : String.valueOf(maxColCount));
    }
```